### PR TITLE
New: Add support of Whatsapp and other social share in Social share block.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Trello - https://trello.com/c/IaswVDGW/138-add-support-of-whatsapp-and-other-social-share-in-social-share-block

### Screenshots

https://share.getcloudapp.com/geuqezNY

### Types of changes

New feature (non-breaking change which adds functionality) 

### How has this been tested?

- Check every new network work well or not.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by vrundakansara — COMMENTED** [review]
Add changelog for this. 👍🏼 Rest all looks okay.

**Comment by vrundakansara** [comment]
@AbhijitNage123 - This would be an improvement, not a new block. Let's add this changelog under Improvement.

File: src/blocks/social-share-child/edit.js [hunk 1]
```diff
@@ -268,6 +276,14 @@ class UAGBSocialShareChild extends Component {
 							{ value: "tumblr", label: __( "Tumblr" ) },
 							{ value: "myspace", label: __( "Myspace" ) },
 							{ value: "email", label: __( "Email" ) },
+							{ value: "pocket", label: __( "Pocket" ) },
+							{ value: "vk", label: __( "VK" ) },
+							{ value: "odnoklassniki", label: __( "Odnoklassniki" ) },
+							{ value: "skype", label: __( "Skype" ) },
+							{ value: "telegram", label: __( "Telegram" ) },
+							{ value: "whatsapp", label: __( "Whatsapp" ) },
```
Comment (vrundakansara) [inline src/blocks/social-share-child/edit.js:284]: It should be `WhatsApp`. Also, check other official names too.

Comment (AbhijitNage123) [reply src/blocks/social-share-child/edit.js]: Done

**Pull request merged**